change-beta/@azure-communication-react-a10e1ac6-f68d-4b21-b2d5-83afb5c7fa04.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "area": "fix",
+  "workstream": "Fixes issue where when local participant stops screen share their local stream would disappear but still be showing to others in the call",
+  "comment": "fix found?",
```

Comment:
nit: I think what you entered for workstream should be entered for comment so that we capture this bug fix comment in the changelog

Reply:
oops fixing